### DISCUSSION TITLE
How to set a different controller/action

### DISCUSSION THREAD

**Discussion by wulffeld** [ddnexus/pagy#536] [Q&A] [answered]
I've searched the issues etc.

I have a front page on a site with pagination where the links should go to /search. This was easy to do with kaminari but I can't figure out how to do this with pagy. https://github.com/kaminari/kaminari#extra-parameters-params-for-the-links

I was doing something like:

```
 params: {controller: "search", action: "index"}
```

with kaminari.

I have other pagination on the site that doesn't need this override. Just the front page.

Similiar issue I found but relates to adding an anchor: https://github.com/ddnexus/pagy/issues/8

**Comment by ddnexus** [comment, marked as answer]
This should solve your case https://ddnexus.github.io/pagy/docs/how-to/#pass-the-request-path

**Reply by ddnexus** [reply]
For the anchor you can use the `:fragment` variable https://ddnexus.github.io/pagy/docs/api/pagy/#other-variables

**Reply by wulffeld** [reply]
It does :) Unbelievable how I missed that trawling the docs. Thanks.
